hexa/pipelines/graphql/schema.graphql

```diff
@@ -36,10 +36,24 @@ enum PipelineRunStatus {
   terminating
 }
 
+enum ParameterTypes {
```

Comment:
The other enums are singular, I think it's better to keep the singular to be consistant.